shell/src/main/java/alluxio/shell/command/FileInfoCommand.java

```diff
@@ -58,12 +57,34 @@ protected void runCommand(AlluxioURI path, CommandLine cl) throws AlluxioExcepti
   }
 
   @Override
+  protected void runCommand(AlluxioURI path, CommandLine cl) throws AlluxioException, IOException {
+    URIStatus status = mFileSystem.getStatus(path);
+    if (status.isFolder()) {
+      List<URIStatus> childrens = mFileSystem.listStatus(path);
```

Comment:
@gjhkael Can you just use `./bin/alluxio fs ls /path/to/dir/` to list the children of a directory?